Automattic/simplenote-ios
# Edit mode/Action Bar updates

### Fix
Fixes #1326 

This PR removes the Action bar from Simplenote iOS.  This has affects the new note button and edit mode.  

Edit mode has been moved into the long press context menu
<img height="400px" src="https://cdn-std.droplr.net/files/acc_593859/rApGoX" />

when in edit mode the new note button is replaced with the Cancel button.  When edit mode is exited the new note button returns.

This also puts the new note button back in the top navigation bar
<img height="400px" src= "https://cdn-std.droplr.net/files/acc_593859/Dyow50" />


### Test
1. Load the app and confirm that in the note list and the editor you don't see the bottom bar and that the new note button is in the top right navigation toolbar
2. Long press on a note from the note list  to see the select option
3. tap on the select option.  Notice the new note button changes to cancel.  
4. Try canceling and select/deleting to make sure the new note button returns after edit mode is exited

### Review
***(Required)*** Add instructions for reviewers.  For example:
> Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release

> `RELEASE-NOTES.txt` was updated in 4e5ad8 with:
> 
> > -   Moved edit mode into a long press action and removed the new note bar #1328
